lib/translator/inquiry.cc

```diff
@@ -266,10 +318,11 @@ void translate(absl::Span<const uint8_t> raw_cmd) {
         // TODO: May optionally be supported by returning Extended INQUIRY data
         // page toapplication client, refer to 6.1.5.
         break;
-      case scsi_defs::PageCode::kBlockLimitsVpd:
-        // TODO: May be supported by returning Block Limits VPD data page to
+      case scsi_defs::PageCode::kBlockLimitsVpd: {
+        // May be supported by returning Block Limits VPD data page to
         // application client, refer to 6.1.6.
-        break;
+        scsi_defs::BlockLimitsVpd result = BuildBlockLimitsVpd();
```

Comment:
why is a variable declared in this scope?

Reply:
to be written to the buffer. handling the completed scsi response is not implemented yet. waiting on public interface.